plugin/src/main/groovy/com/cookpad/android/licensetools/LicenseToolsPlugin.groovy

```diff
@@ -51,6 +51,7 @@ class LicenseToolsPlugin implements Plugin<Project> {
                         message.append("  url: ${libraryInfo.url ?: "#URL#"}\n")
                     }
                     project.logger.warn(message.toString().trim())
+                    appendToLicenseYaml(project, message.toString().trim())
```

Comment:
What I want to solve is that the plugin users forget to create license.yml like #55.
If `checkLicenses` task create it, I think it is simple and convenient for most users.
But I agree with you anyway because `checkLicenses` task should not do more than check.

I will add `updateLicenses` gradle task in another PR. Thank you for the advice.